### PR TITLE
chore: move typing for material design in tns-platform-declarations

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test --ignore

File: tns-core-modules-widgets/build.android.sh [hunk 1]
```diff
@@ -6,32 +6,34 @@ set -e
 echo "Use dumb gradle terminal"
 export TERM=dumb
 
-echo "Clean dist"
-rm -rf dist
-mkdir dist
-mkdir dist/package
-mkdir dist/package/platforms
+rm -rf mkdir dist/package/platforms/android || true
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.android.sh]: `mkdir` should not be here

File: tns-core-modules-widgets/build.android.sh [hunk 2]
```diff
@@ -6,32 +6,34 @@ set -e
...
+mkdir -p dist
+mkdir -p dist/package
+mkdir -p dist/package/platforms
+mkdir dist/package/platforms/android
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.android.sh]: This `mkdir -p` is enough. You can add `-p` here and safely delete the 3 lines above it.

File: tns-core-modules-widgets/build.ios.sh [hunk 3]
```diff
@@ -3,37 +3,57 @@
...
+#     shift # past argument=value
+#     ;;
+#     --skip-pack=*)
+#     SKIP_PACK="true"
+#     shift # past argument=value
+#     ;;
+# esac
+# done
+# echo "PACKAGE_VERSION  = ${PACKAGE_VERSION}"
+# echo "PUBLISH = ${PUBLISH}"
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.ios.sh]: Have you intentionally left this commented code?

File: tns-core-modules-widgets/build.ios.sh [hunk 4]
```diff
@@ -3,37 +3,57 @@
...
-echo "Clean dist"
-rm -rf dist
-mkdir dist
-mkdir dist/package
-mkdir dist/package/platforms
+rm -rf mkdir dist/package/platforms/ios || true
+mkdir -p dist
+mkdir -p dist/package
+mkdir -p dist/package/platforms
+mkdir dist/package/platforms/ios
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.ios.sh]: Only one `mkdir -p` is enough.

File: tns-core-modules-widgets/build.ios.sh [hunk 5]
```diff
@@ -3,37 +3,57 @@
...
+
 echo "Use dumb terminal"
 export TERM=dumb
 
-echo "Clean dist"
-rm -rf dist
-mkdir dist
-mkdir dist/package
-mkdir dist/package/platforms
+rm -rf mkdir dist/package/platforms/ios || true
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.ios.sh]: `mkdir` is redundant.

File: tns-core-modules-widgets/build.sh [hunk 6]
```diff
@@ -3,33 +3,20 @@
...
-cp android/widgets/build/outputs/aar/widgets-release.aar dist/package/platforms/android/widgets-release.aar
-
-echo "Build iOS"
-mkdir dist/package/platforms/ios
-cd ios
-./build.sh
-cd ..
-cp -r ios/TNSWidgets/build/TNSWidgets.framework dist/package/platforms/ios/TNSWidgets.framework
+
+export SKIP_PACK=true
```
Comment (mbektchiev) [inline tns-core-modules-widgets/build.sh]: Maybe forgotten during debugging?

File: tns-core-modules/platforms/ios/src/module.modulemap [hunk 7]
```diff
@@ -1,4 +0,0 @@
-module NativeScriptEmbedder {
```
Comment (mbektchiev) [inline tns-core-modules/platforms/ios/src/module.modulemap]: 😱 Why have you deleted `NativeScriptEmbedder` logic!?

File: tns-core-modules-widgets/build.sh (hunk 6 above)
Comment (SvetoslavTsenov) [reply tns-core-modules-widgets/build.sh]: This one is moved in build.sh for iOS

**Pull request merged**